test/listener_test.py

```diff
@@ -12,6 +12,22 @@
 logging.getLogger("").setLevel(logging.DEBUG)
 
 
+# List of messages of different types that can be used in tests
+TEST_MESSAGES = [
```

Comment:
Nice, we should have had this for a very long time! More specific to BLF though perhaps add a data directory and put an externally created blf file. Eg the test file from the C++ repo: https://bitbucket.org/tobylorenz/vector_blf/raw/3039a27a2cb24bfafb971103e4d61944c4883ca9/src/Vector/BLF/tests/unittests/data/logfile.blf